ArduCopter/GCS_Mavlink.cpp

```diff
@@ -704,6 +704,15 @@ const AP_Param::GroupInfo GCS_MAVLINK::var_info[] = {
     // @Increment: 1
     // @User: Advanced
     AP_GROUPINFO("ADSB",   9, GCS_MAVLINK, streamRates[9],  5),
+
+    // @Param: STATEINFO
+    // @DisplayName: STATEINFO stream rate to ground station
```

Comment:
It just enables / disables the stream, but doesn't change the stream rate right?

Reply:
Correct (the stream rate is dictated by the rate of the streaminfo thread)

Reply:
So @DisplayName and @Description are a little bit confusing

Reply:
Yup, good call